src/app/@ansyn/ansyn/modules/filters/components/boolean-filter-container/boolean-filter-container.component.ts

```diff
@@ -17,9 +17,6 @@ export class BooleanFilterContainerComponent {
 	@Input() counters: BooleanFilterCounters;
 	@Output() onMetadataChange = new EventEmitter<BooleanFilterMetadata>(true);
 
-	@HostBinding('class.rtl')
```

Comment:
remove HostBinding import please

Reply:
HostBinding is still in use, so cannot be removed